scripts/setupgame/setupgame.gml

```diff
@@ -2,108 +2,192 @@
 // https://help.yoyogames.com/hc/en-us/articles/360005277377 for more information
 function SetupGame()
 {
-	if (room != rStageArcade)
+	switch global.gameMode 
 	{
-		p1 = instance_create_layer(104, 104, "Instances", global.p1SelectedCharacter);
-		p2 = instance_create_layer(216, 104, "Instances", global.p2SelectedCharacter);
-	} 
-	else
-	{
-		p1 = instance_create_layer(32, 104, "Instances", global.p1SelectedCharacter);
-		p2 = instance_create_layer(128, 104, "Instances", global.p2SelectedCharacter);
-	}
+		case GAMEMODE.VERSUS:
+		{
+			if (room != rStageArcade)
+			{
+				p1 = instance_create_layer(104, 104, "Instances", global.p1SelectedCharacter);
+				p2 = instance_create_layer(216, 104, "Instances", global.p2SelectedCharacter);
+			} 
+			else
+			{
+				p1 = instance_create_layer(32, 104, "Instances", global.p1SelectedCharacter);
+				p2 = instance_create_layer(128, 104, "Instances", global.p2SelectedCharacter);
+			}
 	
-	// Setup Camera
-	global.camObj = instance_create_layer(80, 0, "Instances", oCamera);
-	global.camObj.p1 = p1;
-	global.camObj.p2 = p2;
+			// Setup Camera
+			global.camObj = instance_create_layer(80, 0, "Instances", oCamera);
+			global.camObj.p1 = p1;
+			global.camObj.p2 = p2;
 
-	ControllerSetup();
+			ControllerSetup();
 	
-	p1.controller = global.player1ControllerSlot;
-	p2.controller = global.player2ControllerSlot;
+			p1.controller = global.player1ControllerSlot;
+			p2.controller = global.player2ControllerSlot;
 
-	p1.playerID = 1;
-	p2.playerID = 2;
+			p1.playerID = 1;
+			p2.playerID = 2;
 	
-	global.currentRound++;
+			global.currentRound++;
 	
-	with (p2) 
-	{
-		image_xscale *= -1;
-		opponent = other.p1;
-		if global.currentRound = 1 hasPerformedIntro = false;
-		PaletteSetup(global.p2PaletteID, global.RusselPalettes);
-	}
-	with (p1) 
-	{
-		opponent = other.p2;
-		if global.currentRound = 1 hasPerformedIntro = false;
-		PaletteSetup(global.p1PaletteID, global.RusselPalettes);
-	}
+			with (p2) 
+			{
+				image_xscale *= -1;
+				opponent = other.p1;
+				if global.currentRound = 1 hasPerformedIntro = false;

```

Comment:
Not sure how this was missed in the refactor, but this if statement needs to be rewritten to fit the style guide:

```
if (global.currentRound == 1)
{
    hasPerformedIntro = false;
}
```